### DISCUSSION TITLE
VSCode + vue3 + ts 为啥那么卡啊？

### DISCUSSION THREAD

**Discussion by WangJincheng4869** [vuejs/language-tools#4014] [Q&A]
用 ts 其实就是想要它友好的代码提示，就像 Java 那样。但是我的项目非常卡，触发提示要30秒左右。用不到的插件全部都禁用了，但是依旧如此。
![图片](https://github.com/vuejs/language-tools/assets/41162158/88024333-fae6-4ee2-a32d-6df890ee6285)

![动画](https://github.com/vuejs/language-tools/assets/41162158/0fbea91d-6fa7-4354-aecb-997771cf80e1)

**Comment by s3xybun** [comment]
+1 回退1.8可以，还是等更新吧
